### DISCUSSION TITLE
How to style only active region handle?

### DISCUSSION THREAD

**Discussion by Benny739** [katspaugh/wavesurfer.js#3890] [Q&A] [answered]
The region-handle-left-active & right part selectors have been removed. Has anybody found a way to style the region handles of only the active region?

**Comment by JanMisker** [comment, marked as answer]
What I did was adding a `part` to the region element, then manipulate the handles based on that.
So when creating the region, or upon some event:
```
region.element.part.add("active-region")
```
and then in css
```
::part(region active-region) {
   // what you want
}
```

**Reply by Benny739** [reply]
Thx, this works :)
